bundler/spec/dependabot/bundler/update_checker_spec.rb

```diff
@@ -2074,98 +2074,4 @@
       it { is_expected.to eq(false) }
     end
   end
-
-  context "with bundler 2 support enabled" do
```

Comment:
All these specs fail because the v2 version resolver is used and returns errors.